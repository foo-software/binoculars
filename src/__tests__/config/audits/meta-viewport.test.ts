import auditMetaViewport from '../../../config/audits/meta-viewport';

describe('meta-viewport audit', () => {
  it('should match snapshot', () => {
    expect(auditMetaViewport('en')).toMatchSnapshot();
  });
});
