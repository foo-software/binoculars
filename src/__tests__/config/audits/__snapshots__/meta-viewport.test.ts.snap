// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`meta-viewport audit should match snapshot 1`] = `
Object {
  "description": "Disabling zooming is problematic for users with low vision who rely on screen magnification to properly see the contents of a web page. [Learn more](https://web.dev/meta-viewport/).",
  "title": "\`[user-scalable=\\"no\\"]\` is not used in the \`<meta name=\\"viewport\\">\` element and the \`[maximum-scale]\` attribute is not less than 5.",
}
`;
